lib/network_monitor.c

```diff
@@ -1015,14 +1015,36 @@ static void check_default_route(struct nm_addr_info *ai)
          */
         mptcpd_addr_get(ai);
 
-        if (l_netlink_send(ai->nm->rtnl,
-                           RTM_GETROUTE,
-                           0,
-                           &store,
-                           buf - (char *) &store,
-                           handle_rtm_getroute,
-                           ai,
-                           NULL) == 0) {
+	unsigned int result = 0;
+
+#ifdef HAVE_L_NETLINK_MESSAGE_NEW_SIZED
```

Comment:
out of curiosity: do we need to maintain the compatibility with older versions of ell? I guess yes, at least for the CI here using ell from the Ubuntu repo, right?

ELL seems to be moving quite fast, but still good to keep a compatibility with a few recent versions if it is relatively easy I suppose.

Reply:
Right!